### PR TITLE
VM Wizard: Custom RAM field compute offering

### PR DESCRIPTION
**Pull request by utchoang**
Open for https://github.com/apache/cloudstack-primate/issues/597
@rhtyd cc @svenvogel I've completed this task. Please review it. Thanks.
image

### CODE REVIEW

**Comment by rhtyd** [comment]
cc @davidjumani @shwstppr
@blueorangutan package

File: src/views/compute/wizard/ComputeSelection.vue [hunk 1]
```diff
@@ -21,31 +21,21 @@
...
-              <a-col :md="4" :lg="4">
-                <a-input-number
-                  v-model="cpuNumberInputValue"
-                  :formatter="value => `${value}`"
-                  @change="($event) => updateComputeCpuNumber($event)"
-                />
-              </a-col>
+              <a-input-number
+                v-model="cpuNumberInputValue"
+                :formatter="value => `${value}`"
```
Comment (davidjumani) [inline src/views/compute/wizard/ComputeSelection.vue:30]: @utchoang Is this needed ?

**Review by davidjumani — APPROVED** [review]
Minor comment. Otherwise tested. LGTM

File: src/views/compute/wizard/ComputeSelection.vue (hunk 1 above)
Comment (utchoang) [reply src/views/compute/wizard/ComputeSelection.vue]: @davidjumani No need. I've removed it.

**Comment by davidjumani** [comment]
@rhtyd Good to go! Thanks @utchoang

**Comment by rhtyd** [comment]
Ping @shwstppr - can you check review the changes
@davidjumani can you test the cases for (a) normal offering, (b) contrainst offering, (c) uncontr... offering; also test the deploy VM and change service offering for a VM actions.

**Comment by rhtyd** [comment]
@blueorangutan package

**Review by shwstppr — APPROVED** [review]
LGTM apart from some UX issues (not related this PR)

**Comment by shwstppr** [comment]
@rhtyd tried to test deployment with different constrained and unconstrained offerings

**Review by shwstppr — CHANGES_REQUESTED** [review]
Not sure if this is some regression. But tested with default values for a constrained offering on Firefox, deployment failed. Used `const2` on QA server

`http://primate-qa.cloudstack.cloud:8080/client/api/?zoneid=04ccc336-d730-42fe-8ff6-5ae36e141e81&templateid=50459b99-5fe0-11ea-9a56-1e006800018c&serviceofferingid=21333d2b-5e42-47a4-b561-af16b0ea75d5&details[0].memory=1536&affinitygroupids=&iptonetworklist[0].networkid=0a7e1136-5a92-49b9-bf5e-44821f7a7815&command=deployVirtualMachine&response=json`

Reverting my LGTM :facepalm:

**Comment by davidjumani** [comment]
@blueorangutan package

**Comment by davidjumani** [comment]
@utchoang The `ComputeSelection.vue` is imported in `ScaleVM.vue`, and the MB is off. Could you fix that too ?
Screenshot from 2020-08-24 10-45-11

**Comment by davidjumani** [comment]
@blueorangutan package

**Review by davidjumani — APPROVED** [review]
LGTM. ScaleVM fix in #610. Together they work!
@rhtyd Good to go!

**Pull request merged**